label_studio/users/models.py

```diff
@@ -164,8 +165,8 @@ def active_organization_contributed_project_number(self):
         return annotations.values_list('project').distinct().count()
 
     @property
-    def own_organization(self):
-        return Organization.objects.get(created_by=self)
+    def own_organization(self) -> Organization:
```

Comment:
This type should be `Optional[Organization]` because fast_first can return a `None`

Reply:
🤔 yeah I think you're right, only owners have an `own_organization` so for others it will return None (previous to my changes it was using `get` so it would just raise an exception 😬

Reply:
I gotta resuscitate that `mypy` PR soon, mypy would 100% catch this :grin: